src/cart/crt_init.c

```diff
@@ -7,16 +7,16 @@
  * This file is part of CaRT. It implements CaRT init and finalize related
  * APIs/handling.
  */
-
 #include <malloc.h>
 #include <sys/mman.h>
 #include <sys/time.h>
 #include <sys/resource.h>
 #include "crt_internal.h"
 
-struct crt_gdata crt_gdata;
-static volatile int   gdata_init_flag;
+struct crt_gdata	crt_gdata;
+static volatile int	gdata_init_flag;
```

Comment:
(style)  Use of volatile is usually wrong: see Documentation/volatile-considered-harmful.txt